src/main/features/ccj/views/confirmation-ccj.njk

```diff
@@ -17,10 +17,11 @@
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
 
-      <p>{{ t('You requested a County Court Judgment on {{ ccjRequestedAt }}. We’ll contact you within 14 days to tell you whether the judgment has been entered.',
+      <p>{{ t("You requested a County Court Judgment on {{ ccjRequestedAt }}. We'll contact you within 5 days to tell you whether the judgment has been entered.",
```

Comment:
Please use the correct apostrophe and single quotes here. Please revert to the existing code for this.

Reply:
option + shift + ]